### ISSUE TITLE
Assigning users an EXO license without EX hybrid

### ISSUE THREAD

**Issue by Larryngolog** [MicrosoftDocs/microsoft-365-docs#2059]
Hello!

I would like to verify what is the process for users who have AAD sync but do not have Exchange Hybrid. 

We are assigning E3 licenses but without EXO to avoid mailbox being created. In such case are the cloud mailboxes created in the background to store Teams data?
If not will EXO license assignment not cause creation of full-features mailbox that will be accessible to the users? 
If mailbox will be created only after EXO license is assigned, are all messages created before EXO license was assigned lost from eDiscovery or will they appear in these mailboxes after they are created?

Thanks!

---
#### Szczegóły dokumentu

⚠ *Nie edytuj tej sekcji. Jest ona wymagana dla docs.microsoft.com ➟ Dołączanie problemu w usłudze GitHub.*

* ID: ab0bb4d7-d9fa-ae92-d3ca-246147d1434a
* Version Independent ID: 7ba20db3-55e2-3d29-2d6c-6f68a6b577a8
* Content: [Searching cloud-based mailboxes for on-premises users - Microsoft 365 Compliance](https://docs.microsoft.com/pl-pl/microsoft-365/compliance/search-cloud-based-mailboxes-for-on-premises-users?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/search-cloud-based-mailboxes-for-on-premises-users.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/search-cloud-based-mailboxes-for-on-premises-users.md)
* Service: **o365-seccomp**
* GitHub Login: @markjjo
* Microsoft Alias: **markjjo**

**Comment by markjjo** [comment]
@Larryngolog I'll try to parse and answer your questions. The topic that you logged this against is about on-premises users, which I don't think maps to your scenario.

1. I assume you are assigning E3 licenses to cloud-based users. Therefore these users will have an identity in the cloud and 1xN Teams chat data will be discoverable. I assume that you will still have to submit a support request to enable feature. 

2. Correct. If you assign an E3 license but do not assign an EXO license, a "full feature mailbox" will not be provisioned. But not assigning EXO license does not affect  \#1. 

3. Any 1xN Teams chat data associated with a E3 user (without an EXO mailbox) will not be moved to the "full featured" mailbox that is provisioned if you assign an EXO license to that E3 user. But that user's 1xN chat data (created before they were licensed in EXO) should still be discoverable if the feature is enabled.

HTHs

**Comment by Larryngolog** [comment]
Hello @markjjo . I thinks You misunderstood me. In the first sentence I mentioned that users are AAD synced. Let me rephrase my question and describe the situation in details:

- Organization just wants to use Teams, without EXO, mail is on-prem in EX Srv
- Organization is using federated identity - ADFS + AAD Connect
- Organization is just synchronizing attributed required for S4B/Teams as described here https://docs.microsoft.com/en-us/azure/active-directory/hybrid/reference-connect-sync-attributes-synchronized#teams-and-skype-for-business-online
- Users have E3 license assigned, but most of the services are disabled. Only Teams service is assigned from E3 license. EXO service is not assigned to avoid creation of EXO mailbox. License assignment is configured by group-based licensing, so any synchronized user gets the license assigned automatically after sync of on-prem identity to the cloud.

In such case:
1. Will users have their "shadow mailbox" created in current setup and thus 1:N chat will be discoverable, without EXO license assigned?
2. If not, then will adding to AAD connect attributes for EXsrv (especially msExchMailboxGuid as this will indicate to EXO that users have mailbox on prem) cause creation if "shadow mailbox"?
3. If not, then enabling EXO in licensing is required, but this will not cause creation of "full featured mailbox", as EXO will detect that user has mailbox on-prem. Only after these two are met - EXO license assigned and msExchMailboxGuid attribute synchronized - will cause creation of "shadow mailbox"?
4. If 2 or 3 are true, then after "shadow mailboxes" are created all chat data generated before this event will be copied over to "shadow mailboxes" , correct?

Hope this clears this situation a bit.

**Comment by Larryngolog** [comment]
Ok will do. I'll follow up with the findings.
